zh_cn/README.md

```diff
@@ -0,0 +1,1627 @@
+
+# Go Concurrency Guide
+
+This guide is built on top of the some examples of the book `Go Concurrency in Go` and `Go Programming Language`
+
```

Comment:
this is was not translated,  of course respect the original names, like `Go Concurrency in Go` `Go Programming Language`